README.md

```diff
@@ -42,6 +42,14 @@ Save and restart (neo)vim and run `:PlugInstall`.
 
 To configure indentation in `purescript-vim` you can use the following variables:
 
+#### g:purescript_disable_indent
+
+Disable indentation altogether.
+
+```vim
+let g:purescript_indent_case = 1
```

Comment:
This looks like the example for the next section. Should this instead be the below?

```suggestion
let g:purescript_disable_indent = 1
```

Reply:
Whoops, copy-paste strikes again :) Thank you for having a look!